src/pydisagg/ihme/splitter/age_splitter.py

```diff
@@ -372,6 +387,11 @@ def split(
                 )
 
         data_group = data.groupby(self.data.index)
+        if output_type == "total":
+            pop_normalize = False
+        elif output_type == "rate":
+            pop_normalize = True
```

Comment:
Minor suggestions here,

1. I think `output_type` should belongs to `typing.Literal`, and we should error out if `output_type` is not `rate` or `total`
2. Should we call `total` `count`?